CONTRIBUTING.md

```diff
@@ -1,6 +1,19 @@
 # Contributing
 
-If you're thinking about contributing to Patrol, thank you! We appreciate your help.
+If you're thinking about contributing to Patrol, thank you! We appreciate your help. Before you get started, we encourage you to read this document. It should make your contributing experience a bit better.
+
+## Running patrol_cli locally
+
+If you want to work with a local version of patrol_cli, you can pick one of two approaches:
+
+Activate patrol_cli locally by typing `dart pub global activate --source path {path to packages/patrol_cli}`. After that you can use `patrol` command exactly the same as any other pub package. 
```

Comment:
I think that instead of {} you should use []. It has been used like that eg. in patrol --help
`Usage: patrol <command> [arguments]`